frontend/public/locales/fr/apply-child.json

```diff
@@ -511,20 +511,38 @@
     "email-address": "Adresse courriel\u00a0: {{email}}",
     "lang-pref-title": "Langue de préférence",
     "lang-pref-change": "Modifier la langue de préférence",
-    "dental-title": "Accès à l'assurance dentaire",
-    "dental-insurance-title": "Accès à une assurance ou une couverture dentaire",
-    "dental-insurance-change": "Modifier la réponse à «\u00a0accès à l'assurance dentaire\u00a0»",
-    "dental-benefit-title": "Accès à des prestations dentaires gouvernementales",
-    "dental-benefit-change": "Modifier la réponse à «\u00a0soins dentaires gouvernementaux\u00a0»",
-    "dental-benefit-has-access": "Accès à des prestations dentaires gouvernementales\u00a0:",
-    "dental-benefit-has-no-access": "Accès à des prestations dentaires gouvernementales\u00a0:",
     "submit-app-title": "Soumettre votre demande",
     "submit-p-proceed": "En transmettant votre demande, vous consentez à attester de la véracité des informations fournies.",
     "submit-p-false-info": "Si vous fournissez de fausses informations sur votre demande, vous et les autres personnes pour lesquelles vous avez postulé pourriez être exclus du régime.",
     "submit-p-repayment": "Si vous ou les membres de votre famille n'étiez pas admissibles, vous ou les membres de votre famille pourriez devoir rembourser le coût total des soins reçus dans le cadre du régime canadien de soins dentaires.",
+    "continue-button": "Continuer",
+    "back-button": "Retour",
     "submit-button": "Soumettre la demande",
     "exit-button": "Quitter la demande",
+    "yes": "Oui",
+    "no": "Non"
+  },
+  "review-child-information": {
+    "read-carefully": "(FR) Please review this information carefully. The information provided will be used by Sun Life to create your child(ren)'s member account.",
+    "page-title": "(FR) Review child(ren) information",
+    "page-sub-title": "(FR) {{child}} information",
+    "dob-title": "Date de naissance",
+    "dob-change": "Modifier la date de naissance",
+    "is-parent": "(FR) Parent or legal guardian of child",
+    "sin-title": "Numéro d'assurance sociale (NAS)",
+    "sin-change": "Modifier le NAS",
+    "full-name-title": "Nom complet",
+    "full-name-change": "Modifier le nom complet",
+    "dental-title": "{{child}} accès à l'assurance dentaire",
+    "dental-insurance-title": "Accès à une assurance ou une couverture dentaire",
+    "dental-insurance-change": "Modifier la réponse à «\u00a0accès à l'assurance dentaire\u00a0»",
+    "dental-benefit-title": "Accès à des prestations dentaires gouvernementales",
+    "dental-benefit-change": "Modifier la réponse à «\u00a0soins dentaires gouvernementaux\u00a0»",
+    "dental-benefit-has-access": "Accès à des prestations dentaires gouvernementales\u00a0:",
+    "dental-benefit-has-no-access": "Accès à des prestations dentaires gouvernementales\u00a0:",
+    "continue-button": "Continuer",
     "back-button": "Retour",
+    "exit-button": "Quitter la demande",
```

Comment:
That key shouldn't exist in the "child" flow context for "review-child-information".

Reply:
Removed unused translation